Arrays/2779MaximumBeautyofanArrayAfterApplyingOperation.py

```
@@ -0,0 +1,62 @@
+            right=nums[i]-k 
+            if left>=0:
+               if right-left<=2*k:
+                   count+=1
+            else:
+                hash_map[nums[i]]+=1
+        hash_map=list(hash_map.items())
+        hash_map.sort(reverse=True,key= lambda x:x[1])
+        return max(count,hash_map[0][1])       
+        
+Solution().maximumBeauty([43,86,33,18],23)
+class TestApp:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)